INN/umbrella-workdayminn
# Child theme header and navigation styles + updates

## Changes

This pull request makes the following changes:

- New styling for the Workday child theme header navigation.

## Why



For #2 

## Testing/Questions

Features that this PR affects:

- Workday child theme header + nav


Questions that need to be answered before merging:

- [x] Does everything still work ok?
- [x] Are any requirements missing?
- [x] Is there a better way to do any of this?

Steps to test this PR:

1. View desktop size version of the site and verify the header meets redesign requirements
2. View mobile size version of the site and verify it meets redesign requirements
